### ISSUE TITLE
Error creating synthetic script monitor targeting legacy runtime.

### ISSUE THREAD

**Issue by usfdmiller** [newrelic/newrelic-client-go#982]
### Description
Trying to create a newrelic_synthetics_script_monitor resource targeting the legacy (Node 10) runtime results in an error.  

### Go Version
go version go1.19.4 darwin/amd64

### Current behavior
As the description say, trying to create a synthetic script monitor that targets the legacy Node 10 runtime results in an error.

```json
{
 "data": {
  "syntheticsCreateScriptApiMonitor": {
   "errors": [
    {
     "description": "Runtime values are invalid combination.",
     "type": "BAD_REQUEST"
    }
   ],
   "monitor": null
  }
 }
}
```

### Expected behavior
I expect the synthetic script monitor to be deployed without error.  According to the documentation it should still be possible to target the legacy runtime.

> If you want to use a legacy runtime (Node 10 or Chrome 72) you can set the runtime_type, runtime_type_version and script_language to empty string "". 

### Steps To Reproduce

1. Run "terraform apply" with the following configuration.

```hcl
resource "newrelic_synthetics_script_monitor" "monitor" {
  status               = "ENABLED"
  name                 = "script_monitor"
  type                 = "SCRIPT_API"
  locations_public     = ["US_WEST_1"]
  period               = "EVERY_6_HOURS"

  script               = "console.log('it works!')"

  script_language      = ""
  runtime_type         = ""
  runtime_type_version = ""

  tag {
    key    = "some_key"
    values = ["some_value"]
  }
}
```

### Debug Output (if applicable)
Looking the debug output for the request, the script_language field is not being sent in the API payload.

```json
{
 "query": "mutation(\n\t$accountId: Int!,\n\t$monitor: SyntheticsCreateScriptApiMonitorInput!,\n) { syntheticsCreateScriptApiMonitor(\n\taccountId: $accountId,\n\tmonitor: $monitor,\n) {\n\terrors {\n\t\tdescription\n\t\ttype\n\t}\n\tmonitor {\n\t\tcreatedAt\n\t\tguid\n\t\tid\n\t\tlocations {\n\t\t\tprivate\n\t\t\tpublic\n\t\t}\n\t\tmodifiedAt\n\t\tname\n\t\tperiod\n\t\truntime {\n\t\t\truntimeType\n\t\t\truntimeTypeVersion\n\t\t\tscriptLanguage\n\t\t}\n\t\tstatus\n\t}\n} }",
 "variables": {
  "accountId": 123456,
  "monitor": {
   "locations": {
    "public": [
     "US_WEST_1"
    ]
   },
   "name": "script_monitor",
   "period": "EVERY_6_HOURS",
   "runtime": {
    "runtimeType": "",
    "runtimeTypeVersion": ""
   },
   "script": "console.log('it works!')",
   "status": "ENABLED",
   "tags": [
    {
     "key": "some_key",
     "values": [
      "some_value"
     ]
    }
   ]
  }
 }
}
```

### Additional Context
This has not been a problem until _very_ recently since I was deploying synthetic monitors targeting the legacy runtime on Monday or Tuesday of this week.  I suspect there was a validation change made to the API.  

### References or Related Issues
This is probably the result of another "omitempty" flag being set on a struct.  Probably this one right here:
https://github.com/newrelic/newrelic-client-go/blob/main/pkg/synthetics/types.go#L811

**Comment by sanderblue** [comment]
Hi @usfdmiller This looks like a breaking change was introduced in the API. We'll likely need to adjust accordingly based on your feedback.

**Comment by sanderblue** [comment]
Closing. Fixed in #983
